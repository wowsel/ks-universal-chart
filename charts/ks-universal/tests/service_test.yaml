suite: test service
templates:
  - service.yaml
tests:
  - it: should create a service with specified configuration
    set:
      services:
        test-service:
          type: ClusterIP
          ports:
            - name: http
              port: 80
              targetPort: 8080
            - name: https
              port: 443
              targetPort: 8443
    asserts:
      - isKind:
          of: Service
      - isAPIVersion:
          of: v1
      - equal:
          path: metadata.name
          value: test-service
      - equal:
          path: spec.type
          value: ClusterIP
      - matchSnapshot:
          path: spec.ports 