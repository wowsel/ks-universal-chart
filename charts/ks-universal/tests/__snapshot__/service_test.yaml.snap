should create a service with specified configuration:
  1: |
    - name: http
      port: 80
      protocol: TCP
      targetPort: 8080
    - name: https
      port: 443
      protocol: TCP
      targetPort: 8443
